### PR TITLE
Added first solution to Project Euler problem 135

### PR DESCRIPTION
**Pull request by sharmapulkit04**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by sharmapulkit04** [comment]
Hi,@dhruvmanila and @Kush1101, I created a directory of problem 135 of the Project Euler project and added first solution .
This pull request is in refrence to the ongoing hacktoberfest2020.

**Review by 53jk1 — APPROVED** [review]

File: project_euler/problem_135/sol1.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            else:
+                d /= 4
+                if a > d and a < 4 * d:  # since x,y,z are positive integers
+                    freq[n] += 1  # so z>0 and a>d ,also 4d<a
+    count = 0
+    for n in range(1, limit):
+        if freq[n] == 10:
+            count += 1
+
+    return count
```
Comment (dhruvmanila) [inline project_euler/problem_135/sol1.py:56]: Please add the `if __name__ == "__main__":` block as mentioned in project_euler/README.md

File: project_euler/problem_135/sol1.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    have exactly ten distinct solutions.
+    >>> solution(100)
+    0
+    >>> solution(10000)
+    45
+    >>> solution(50050)
+    292
+    """
+    limit = limit + 1
+    freq = [0] * limit
```
Comment (dhruvmanila) [inline project_euler/problem_135/sol1.py:41]: Descriptive variable names than `freq`, `a`, `n`, `d`

File: project_euler/problem_135/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+#
```
Comment (dhruvmanila) [inline project_euler/problem_135/__init__.py:1]: No need for anything in the `__init__.py` file. This file is supposed to be empty.

**Review by dhruvmanila — CHANGES_REQUESTED** [review]

File: project_euler/problem_135/sol1.py (hunk 2 above)
Comment (sharmapulkit04) [reply project_euler/problem_135/sol1.py:41]: can change freq to frequency but a and d as mentioned in my solution approach are standard ways of taking the first term and common difference in an A.P and n is mentioned in the question.

File: project_euler/problem_135/sol1.py (hunk 1 above)
Comment (sharmapulkit04) [reply project_euler/problem_135/sol1.py:56]: And also adding  print(f"{solution() = }") ,the equal sign at the end is causing my pull request to fail at black tests, I am new pre-commit hooks and open source development. Need help.

File: project_euler/problem_135/sol1.py (hunk 2 above)
Comment (dhruvmanila) [reply project_euler/problem_135/sol1.py:41]: Yes for `freq`.
I don't know what does A.P. means but as we're coding in Python and also there are some standards we follow in this repository to improve the readability and maintainability of code, we require descriptive variable names.

Why? This code might be looked by thousands of other people and they might wonder what does `a` or `d` even mean? They would have to do some googling or what not. So, maybe it is better to provide a name which is self explanatory :)

File: project_euler/problem_135/sol1.py (hunk 1 above)
Comment (dhruvmanila) [reply project_euler/problem_135/sol1.py:56]: Yes

**Review by dhruvmanila — CHANGES_REQUESTED** [review]

**Pull request closed without merging**